### PR TITLE
Client maktub develop ruan

### PR DESCRIPTION
**Pull request by Flame1994**

### CODE REVIEW

**Comment by Flame1994** [comment]
Added a few new sections and mostly finished up the layout and templates.

File: src/App.vue [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
+		<v-row>
+			<v-col cols="10" class="offset-1">
+				<div style="position:relative;">
+					<app-bar logo="header-logo-white.png" :links="headerLinks"></app-bar>
+				</div>
+			</v-col>
+		</v-row>
 		<v-main>
-			<section-landing title="Provided Page Title" background-image="https://i.imgur.com/bX2Uk7C.jpg">
+			<section-landing title="SEARCH AND YOU SHALL FIND. DON’T MISS MAKTUB" background-image="/maktub-banner.png">
```
Comment (angelomelonas) [inline src/App.vue]: Please move all the images to the `/assets/image` directory and then use them like this: 

`:background-image="require(`@/assets/image/maktub-banner.png`)"`

That way we can easily swap out images by making them variables.

**Review by angelomelonas — APPROVED** [review]
Looks really good. Just the one small comment.

File: src/App.vue (hunk 1 above)
Comment (Flame1994) [reply src/App.vue]: Ah sure, doing that quickly.

Comment (Flame1994) [reply src/App.vue]: @angelomelonas Completed. I implemented the same concept as with `AppBar` and `FooterSimple`. You pass through the name of the image, which is used as a prop in the child component. So in the child component it is used like `:src="require(@/assets/image/${image})"`

**Comment by Flame1994** [comment]
@angelomelonas Merged. Pull latest then you should see all the updated images, etc. There are some still missing, so feel free to add whatever you like there.

**Pull request merged**